spec/index.md

```diff
@@ -5116,6 +5216,22 @@ S with
 
 ```
 
+#### Dropping expired messages {#message-timeout}
```

Comment:
Why not drop expired messages in their expiry transition instead of setting the deadline to expired and dropping them in a separate transition?

Reply:
Or we do need to keep expired messages around since they can still have side-effects on the callee?

Reply:
The callee may still process a `CallMessage` even though it's expired. Furthermore, the callee may produce a response message even though the caller has already received a `SYS_UNKNOWN`.